Broadcom/symantec-endpoint-protection/_meta/fields.yml

```
@@ -0,0 +1,156 @@
+threat.enrichments:
+  name: threat.enrichments
+  description: ""
+  type: array
+
+threat.enrichments.indicator:
+  name: threat.enrichments.indicator
+  description: ""
+  type: dict
```

```suggestion
  type: object
```

`object` in ECS (I am not sure if it works in Ingest)